src/Codeception/Lib/InnerBrowser.php

```diff
@@ -295,6 +308,9 @@ private function getCrawler()
         return $this->crawler;
     }
 
+    /**
+     * @return \Symfony\Component\BrowserKit\AbstractBrowser
```

Comment:
This class was named `Symfony\Component\BrowserKit\Client` before Symfony 4.3, we have class_alias at the top of this file to support older versions.
I don't know if it is necessary to add old name to this annotation.